sdk/tables/Azure.Data.Tables/tests/TableEntityTests.cs

```diff
@@ -59,14 +59,27 @@ public void DynamicEntityGetWrongTypesThrows()
             Assert.That(() => entityWithAllTypes.GetInt64("binary"), Throws.InstanceOf<InvalidOperationException>(), "GetInt64 should validate that the value for the inputted key is a Int64.");
         }
 
+        /// <summary>
+        /// Validates getters for non-nullable types involving null. (ex. DateTime, int)
+        /// </summary>
+        [Test]
+        public void ValidateDictionaryTableEntityGetNonNullType()
```

Comment:
Wasn't the intent of this test to validate using `Get*` for a type matched key that holds a null value?

Reply:
This test was to make sure that `Get*` for non-null types like value types of structs would still return null when the inputted key had never been set. Before, calling `TableEntity.Timestamp` on a newly created `TableEntity` would throw an error because it was trying to cast `null` as a `DateTimeOffset`.

But that's probably a test that I should write!